it/controllers/pensions/annualAllowance/ReducedAnnualAllowanceTypeControllersControllerISpec.scala

```diff
@@ -0,0 +1,639 @@
+            urlGet(fullUrl(reducedAnnualAllowanceTypeUrl(taxYearEOY)), user.isWelsh, follow = false,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption(taxYearEOY), captionSelector)
+          hintTextCheck(checkboxHint, Selectors.checkboxHintSelector)
+          checkBoxCheck(checkboxMoneyPurchaseText, 1, checked = Some(true))
+          checkBoxCheck(checkboxTaperedText, 2, checked = None)
+          inputFieldValueCheck("reducedAnnualAllowanceType[]", Selectors.checkboxMoneyPurchaseSelector, moneyPurchaseCheckboxValue)
+          inputFieldValueCheck("reducedAnnualAllowanceType[]", Selectors.checkboxTaperedSelector, taperedCheckboxValue)
+
+          textOnPageCheck(expectedDetailsRevealText, Selectors.expectedDetailsRevealTextSelector)
+          textOnPageCheck(expectedDetailsMoneyPurchaseText, Selectors.detailsParagraphSelector(1))
+          textOnPageCheck(user.specificExpectedResults.get.expectedDetailsMoneyParagraphText, Selectors.detailsParagraphSelector(2))
+          textOnPageCheck(expectedDetailsBullet1, Selectors.bulletSelector(1))
+          textOnPageCheck(expectedDetailsBullet2, Selectors.bulletSelector(2))
+          textOnPageCheck(expectedDetailsTaperedText, Selectors.detailsParagraphSelector(4))
+          textOnPageCheck(user.specificExpectedResults.get.expectedDetailsTaperedParagraphText, Selectors.detailsParagraphSelector(5))
+          linkCheck(expectedDetailsExternalLinkText,
+            expectedDetailsLinkSelector, externalHref)
+
+          buttonCheck(expectedButtonText, continueButtonSelector)
+          formPostLinkCheck(reducedAnnualAllowanceTypeUrl(taxYearEOY), formSelector)
+          welshToggleCheck(user.isWelsh)
+        }
+      }
+    }
+
+    "redirect to the reduced annual allowance question page if it has not been answered" which {
+
+      implicit lazy val result: WSResponse = {
+        authoriseAgentOrIndividual(isAgent = false)
+        dropPensionsDB()
+        val pensionsViewModel = aPensionAnnualAllowanceViewModel.copy(
+          reducedAnnualAllowanceQuestion = None,
+          moneyPurchaseAnnualAllowance = None,
+          taperedAnnualAllowance = None
+        )
+
+        insertCyaData(pensionsUserDataWithAnnualAllowances(pensionsViewModel), aUserRequest)
+        urlGet(fullUrl(reducedAnnualAllowanceTypeUrl(taxYearEOY)), follow = false,
+          headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+      }
+
+      "has an SEE_OTHER status" in {
+        result.status shouldBe SEE_OTHER
+        //TODO - go to annual allowance cya page when available
```

Comment:
TODO should be "Do you have a reduced annual allowance?" page

same for the next test as well